Tests/TuistKitTests/Services/Cache/CacheProfileResolverTests.swift

```diff
@@ -0,0 +1,125 @@
+import Foundation
+import TSCBasic
+import TuistCacheTesting
+import TuistGraph
+import TuistSupport
+import XCTest
+
+@testable import TuistKit
+@testable import TuistSupportTesting
+
+final class CacheProfileResolverTests: TuistUnitTestCase {
+    private typealias ResolvedCacheProfile = CacheProfileResolver.ResolvedCacheProfile
+
+    var subject: CacheProfileResolver!
+
+    override func setUp() {
+        super.setUp()
+        subject = CacheProfileResolver()
+    }
+
+    override func tearDown() {
+        subject = nil
+        super.tearDown()
+    }
+
+    func testItResolvesDefaultProfileFromTuistDefaults() {
```

Comment:
nitpick: most of words in a name of test cases are delimited by underscore rather than using camel case